src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let men = people.filter((person) => person.sex === 'm');
+
+  if (century) {
+    men = men.filter((person) => Math.ceil(person.died / 100) === century);
+  }
```

Comment:
You can combine `sex` and `century` filters in one array loop instead of two. Imagine you have a huge array, so these will be very expensive operations.